src/Grpc.Net.Client/Internal/StreamExtensions.cs

```diff
@@ -156,12 +157,18 @@ private static Status CreateUnknownMessageEncodingMessageStatus(string unsupport
                         throw new RpcException(CreateUnknownMessageEncodingMessageStatus(grpcEncoding, GrpcProtocolConstants.CompressionProviders.Select(c => c.EncodingName)));
                     }
 
+#pragma warning disable CS8600 // Converting null literal or possible null value to non-nullable type.
```

Comment:
Nullable attribute isn't working here for some reason. Will clean it up when bug in compiler is fixed. Only effects designer.